### PR TITLE
add setBindingDebugInfo method, with support for template placeholder comments

### PR DESCRIPTION
**Pull request by jeffbcross**

### CODE REVIEW

**Comment by jeffbcross** [comment]
Travis is happy, I'm going to test this internally.

File: modules/angular2/src/platform/dom/dom_renderer.ts [hunk 1]
```diff
@@ -41,11 +43,13 @@ import {
 import {camelCaseToDashCase} from './util';
 import {ViewEncapsulation} from 'angular2/src/core/metadata';
 
-// TODO move it once DomAdapter is moved
+// TODO move it once DdomAdapter is moved
```
Comment (jeffbcross) [inline modules/angular2/src/platform/dom/dom_renderer.ts]: Ugh

**Comment by jeffbcross** [comment]
@tbosch mind taking a look?

**Comment by jeffbcross** [comment]
Needs some work after running internal tests.

**Comment by jeffbcross** [comment]
I'm focusing on another higher-priority beta issue regarding #5036 right now, will come back to this one once that's resolved.

File: modules/angular2/src/platform/dom/dom_renderer.ts [hunk 2]
```diff
@@ -146,6 +150,31 @@ export abstract class DomRenderer extends Renderer implements NodeFactory<Node>
...
+    var element = view.boundElements[location.boundElementIndex];
+    var dashCasedPropertyName = camelCaseToDashCase(propertyName);
+    if (DOM.isCommentNode(element)) {
+      var existingBindings = RegExpWrapper.firstMatch(
+          TEMPLATE_BINDINGS_EXP, StringWrapper.replaceAll(DOM.getText(element), /\n/g, ''));
+      var parsedBindings = Json.parse(existingBindings[1]);
+      if (isPresent(propertyValue)) {
+        parsedBindings[dashCasedPropertyName] = propertyValue;
+      } else {
+        StringMapWrapper.delete(parsedBindings, dashCasedPropertyName);
```
Comment (vsavkin) [inline modules/angular2/src/platform/dom/dom_renderer.ts]: Remove this

**Comment by IgorMinar** [comment]
@jeffbcross @vsavkin can we get this one in? or should we punt it to beta.1?

**Comment by jeffbcross** [comment]
This landed in a separate PR in an inception-like fashion.

**Pull request closed without merging**